policies_examples.md

```diff
@@ -0,0 +1,178 @@
+            <demand-adjust fillout="1" year="1975">21</demand-adjust>
+            <market>global</market>
+            <linked-policy>GHG</linked-policy>
+            <price-unit>1990$/GgCH4</price-unit>
+	...
+```
+
+## <a name="energy-constraint"> Energy Constraint </a>
+The following inputs will set a constraint on bioenergy use in the USA, limiting it to 10 EJ/yr. Note that the `input-tax` tag will need to be added to all model periods (the example only uses 2020 for brevity). Additionally, this tag needs to be added to all production technologies and regions that are included in the target. For example, if you wanted to include 1st generation bioenergy in this constraint, you would need to add a tag to those technologies in the refinery sector, as 1st generation bioenergy does not go through the "regional biomass" market. If you wanted to only constrain one type of bioenergy, then would only put the `input-tax` tag in the technology producing that type of bioenergy (e.g., you could include a technology with this `input-tax` in the biomass resource to limit MSW production only). This example sets an upper bound on production. If instead you wanted a lower bound, then you would use `input-subsidy` in the technology and `policyType` equals "subsidy" in the policy-portfolio-standard. If you wanted to set an exact constraint, you can use either a tax or a subsidy with the additional tag `<min-price year="2020" fillout="1">-100</min-price>` which will allow the tax or subsidy to go negative, effectively enabling either a tax or a subsidy within the same constraint. 
+
+```
+<scenario>
+   <world>
+      <region name="USA">
+         <supplysector name="regional biomass">
+            <subsector name="regional biomass">
+               <technology name="regional biomass">
+                  <period year="2020">
+                     <input-tax name="bio-constraint"/>
+                  </period>
+                  ...
+               </technology>
+            </subsector>
+        </supplysector>
+        <policy-portfolio-standard name="bio-constraint">
+				<market>USA</market>
+				<policyType>tax</policyType>	
+				<constraint year="2020" fillout="1">10</constraint>
+		</policy-portfolio-standard>
+    </region>
+	</world>
+</scenario>
+```
+
+## <a name="land-constraint"> Land Constraint </a>
+The following input file will keep UnmanagedForest area in the USA GreatLakes region above 12 thous sq km, starting in the year 2020 and going through out the model time horizon. This could be converted to an upper bound by changing `policyType` to "tax". Note that the `land-constraint-policy` tag will need to be added to any `UnmanagedLandLeaf` you want to constraint. If more than one `UnmanagedLandLeaf` is given the same policy name ("reduced_deforestation" in this example) and market name (USA in this example), then the sum of those leafs will be constrained to the specified amount.
```

Comment:
Could add some additional explanation here, e.g.:

The following input file will keep UnmanagedForest area in the USA GreatLakes region above 12 thous sq km, starting in the year 2020 and  throughout the model time horizon. Because the policy type is specified as `<policyType>subsidy</policyType>` the model will add a subsidy to the associated  market to achieve the specified 12 thous sq km target.  

If an upper bound was needed instead, this can be implemented by changing `policyType` to "tax" in the below xml example. In this case the opposite will occur, with the model adding a tax to the associated market to keep land use below the specified value.